calibration/NUTNRB/CGINS-NUTNRB-00239__20160201.csv

```diff
@@ -0,0 +1,8 @@
+serial,name,value,notes,
```

Comment:
This one was added per SW review comment for CGINS-NUTNRB-00239__20160624.csv which was not needed and deleted, and this one was created.